### DISCUSSION TITLE
[Feature]: make FilamentPanel inside Module

### DISCUSSION THREAD

**Discussion by HelenaGwyn** [savannabits/filament-modules#134] [Ideas]
### What happened?

I would like to create extra panels inside a Module instead of using it as plugin in a panel.
I saw this in a fork that isn't finished I assume https://github.com/laraxot/filament-modules.

### How to reproduce the bug

There is no command to generate a panel.

### Package Version

3

### PHP Version

8.3

### Laravel Version

11

**Comment by coolsam726** [comment]
Check #137
